bodhi/tests/server/test_renderers.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2017 Red Hat, Inc.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import copy
+import re
+
+from webtest import TestApp
+
+from bodhi.server import main
+import bodhi.tests.server.functional.base
+
+
+class TestRenderers(bodhi.tests.server.functional.base.BaseWSGICase):
+
+    def test_renderer_jpeg(self):
+        """
+        Test that the renderer returns a jpeg. In this case, the CAPTCHA image.
+        """
+
+        settings = copy.copy(self.app_settings)
+        settings.update({
+            'authtkt.secret': 'whatever',
+            'authtkt.secure': True,
+            'captcha.secret': '2o78T5zF7OERyAtBfC570ZX2TXvfmI3R5mvw6LkG3W0=',
+            'captcha.image_width': '300',
+            'captcha.image_height': '80',
+            'captcha.font_path': '/usr/share/fonts/liberation/LiberationMono-Regular.ttf',
+            'captcha.font_size': '36',
+            'captcha.font_color': '#000000',
+            'captcha.background_color': '#ffffff',
+            'captcha.padding': '5',
+            'captcha.ttl': '300',
+        })
+        app = TestApp(main({}, session=self.db, **settings))
+
+        res = app.get('/updates/FEDORA-2017-a3bbe1a8f2', headers=dict(accept='text/html'))
+        captcha_url = re.search(r'"http://localhost(/captcha/[^"]*)"', str(res)).groups()[0]
+        resp = app.get(captcha_url)
+        self.assertIn('image/jpeg', resp.headers['Content-Type'])
```

Comment:
We could probably use Pillow to assert that the image is a valid JPEG. Perhaps we could use [```pillow```](https://pillow.readthedocs.io/en/4.1.x/) to load the image and assert that it's is 300x80 pixels?

I'd say this is optional if you don't feel like doing it right now - we can always do it later.